src/deploy.ts

```diff
@@ -318,7 +318,8 @@ export class MBDeployer implements MBDeployerI {
    */
   private async linkContractToAddress(
     contract: MultiBaasContract,
-    address: MultiBaasAddress
+    address: MultiBaasAddress,
+    startingBlock: string
```

Comment:
(FYI, I have used [this sample project](https://github.com/curvegrid/multibaas-sample-hardhat-project) for test.)

According to [here](https://github.com/curvegrid/hardhat-multibaas-plugin/pull/25/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R150), an empty string should disable (I am not sure what disabled message in UI) but it looks event monitor running.

<img width="277" alt="empty-string" src="https://user-images.githubusercontent.com/16159767/174520650-0ef367c2-69c5-41e5-8510-2fe389c9e9a3.png">

And I am not sure what the differences between `undefined`, `"0"`, `"-1000"`, `"12"` in the MultiBaas UI, and all same as below. (Lack of context, please teach me :bow:)

_After deploy()_
<img width="285" alt="undefined,-10000, 0, 15" src="https://user-images.githubusercontent.com/16159767/174520655-9eba1366-7db0-48cc-8d5f-8fe6db8103b5.png">

_After deployProxy()_
<img width="607" alt="image" src="https://user-images.githubusercontent.com/16159767/174520686-df7668a9-839d-454c-989f-93862a2ada66.png">

Reply:
Hmmm...good call out. Not sure if this is because the UI has not been updated to work with the new endpoint yet or not.